frontend/src/components/LandingPage/Weather/Weather.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Weather.css'
+
+const weatherData = {
+    city: "Allen" + " ",
+    temperature: "65" + '\xB0F ',
+    picture: "/partly cloudy.png"
+};
+
+class Weather extends React.Component {
```

Comment:
There are no state changes, so it is best to make this component a functional component instead of a class component. I recommend using functional component when there are no state changes and a class component when there are state changes.

Reply:
Right now the json object values are hardcoded in, but when the API call is coded then the values will change. Should I still make it a functional component?